### DISCUSSION TITLE
Trieve Functionality.

### DISCUSSION THREAD

**Discussion by AjGaht** [yournextstore/yournextstore#77] [Q&A]
Hi everyone,
Making Trieve work with stripe seems tricky for me. 
I'd really appreciate some support on how to set up and handle data from stripe.

At the moment, this my error :
Console Error
[ Server ] Error: HTTP error! status: {"message":"No positive or negative chunk ids could be found. At least one positive or negative id must be provided."}

**Comment by zaiste** [comment]
@AjGaht hi!

You need to create an index in Trieve and upload data there specyfing the fields you want to use for the search

**Reply by AjGaht** [reply]
@zaiste Hello!
Is there a template or guide you can share in relation to (Your Next Store), on how to create an index in Trieve and the type of Data to upload.
Thanks.

**Reply by AjGaht** [reply]
@zaiste , A quick one,

I figured out the indexing and data upload to Trieve, but now, suggestions are only showing on my local development but once i deploy to vercel it's not showing. 
Error :
The resource https://trendz-up.vercel.app/_next/image?url=https%3A%2F%2Ffiles.stripe.com%2Flinks%2FMDB8YWNjdF8xUUxPQVNHRVVVbXJjYlVQfGZsX3Rlc3RfMXZxQVBTTXFoUWtNY3FTUmJtaXRvcXVv00QJNOkXzu&w=3840&q=75 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.

Thanks for your time and support🙏
